src/sentry/api/endpoints/group_ai_autofix.py

```diff
@@ -0,0 +1,170 @@
+
+        release: Release = Release.objects.get(version=release_version)
+        release_commits: list[ReleaseCommit] = ReleaseCommit.objects.filter(release=release)
+
+        commits: list[Commit] = [release_commit.commit for release_commit in release_commits]
+        base_commit: Commit | None = None
+        for commit in commits:
+            repo: Repository = Repository.objects.get(id=commit.repository_id)
+            # Hardcoded to only accept getsentry/sentry repo for now
+            if repo.external_id == "getsentry/sentry":
+                base_commit = commit
+                break
+
+        if not base_commit:
+            metadata["autofix"] = {
+                **metadata["autofix"],
+                "completedAt": datetime.now().isoformat(),
+                "status": "ERROR",
+                "fix": None,
+                # Hardcoded to only accept getsentry/sentry repo for now
+                "errorMessage": "No valid base commit found for release; only getsentry/sentry repo is supported right now.",
+            }
+            group.data["metadata"] = metadata
+            group.save()
+
+            return HttpResponse(
+                status=400,
+            )
+
+        try:
+            seer_connection_pool.urlopen(
+                "POST",
+                "/v0/automation/autofix",
+                body=json.dumps(
+                    {
+                        "base_commit_sha": base_commit.key,
+                        "issue": {
+                            "id": str(group.id),
+                            "title": group.title,
+                            "events": [event_stacktrace],
+                        },
+                        "additional_context": data.get("additional_context", ""),
+                    }
+                ),
+                headers={"content-type": "application/json;charset=utf-8"},
+            )
+
+            # Mark the task as completed after TIMEOUT_SECONDS
+            check_for_timeout.apply_async(
+                kwargs={
+                    "group_id": group.id,
+                    "created_at": created_at,
+                },
+                countdown=TIMEOUT_SECONDS,
+            )
+        except Exception as e:
+            metadata["autofix"] = {
+                **metadata["autofix"],
+                "completedAt": datetime.now().isoformat(),
+                "status": "ERROR",
+                "fix": None,
+                "errorMessage": "Failed to send autofix to seer.",
+            }
+
+            logger.exception(
+                "Failed to send autofix to seer",
+                extra={
+                    "group_id": group.id,
+                    "created_at": created_at,
+                    "exception": e,
+                },
+            )
+
+        group.data["metadata"] = metadata
+
+        group.save()
+
+        return HttpResponse(
+            status=202,
+        )
+
+    def get(self, request: Request, group: Group) -> HttpResponse:
```

Comment:
@jennmueng do we even need this route? Feels like you could just query the issue itself.

Reply:
Originally was querying the issue itself but because this route will be polled while the autofix is processing I think it made sense to have a specific route that returns just the autofix